src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/MemberRefMetadataDecoder.vb

```diff
@@ -114,9 +114,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
                             Return Nothing
                         End If
 
-                        Dim customModifiers As ImmutableArray(Of ModifierInfo(Of TypeSymbol)) = Nothing
-                        Dim type As TypeSymbol = Me.DecodeFieldSignature(signaturePointer, customModifiers)
-                        Return FindFieldBySignature(_containingType, memberName, customModifiers, type)
+                        Dim fieldInfo As FieldInfo(Of TypeSymbol) = Me.DecodeFieldSignature(signaturePointer)
+                        Return FindFieldBySignature(_containingType, memberName, fieldInfo.CustomModifiers, fieldInfo.Type)
```

Comment:
>FindFieldBySignature(_containingType, memberName, fieldInfo.CustomModifiers, fieldInfo.Type) [](http://example.com/codeflow?start=31&length=92)

It doesn't look right to ignore 'ref' here. Perhaps it is fine to not find a match for a ref field because VB doesn't support ref fields, but ignoring the presence of the 'ref' feels wrong.

Reply:
Added a test and PROTOTYPE comments for now.